docs/readthedocs/source/doc/Nano/QuickStart/pytorch_lightninglite.md

```diff
@@ -0,0 +1,143 @@
+
+```python
+from bigdl.nano.pytorch.vision import transforms
+from bigdl.nano.pytorch.vision.datasets import CIFAR10
+```
+
+### **Step 2: Load the Data**
+
+Import Cifar10 dataset from torch_vision and modify the train transform. You could access [CIFAR10](https://www.cs.toronto.edu/~kriz/cifar.html) for a view of the whole dataset.
+
+```python
+from torch.utils.data import DataLoader
+
+from bigdl.nano.pytorch.vision import transforms
+from bigdl.nano.pytorch.vision.datasets import CIFAR10
+
+def create_dataloader(data_path, batch_size):
+    train_transform = transforms.Compose([
+        transforms.Resize(256),
+        transforms.ColorJitter(),
+        transforms.RandomCrop(224),
+        transforms.RandomHorizontalFlip(),
+        transforms.Resize(128),
+        transforms.ToTensor()
+    ])
+
+    train_dataset = CIFAR10(root=data_path, train=True,
+                            download=True, transform=train_transform)
+
+    train_loader = DataLoader(train_dataset, batch_size=batch_size,
+                              shuffle=True, num_workers=0)
+
+    return train_loader
+```
+
+### **Step 3: Define the Model**
+
+You may define your model in the same way as the standard PyTorch models.
+
+```python
+from torch import nn
+
+from bigdl.nano.pytorch.vision.models import vision
+
+class ResNet18(nn.Module):
+    def __init__(self, num_classes, pretrained=True, include_top=False, freeze=True):
+        super().__init__()
+        backbone = vision.resnet18(pretrained=pretrained, include_top=include_top, freeze=freeze)
+        output_size = backbone.get_output_size()
+        head = nn.Linear(output_size, num_classes)
+        self.model = nn.Sequential(backbone, head)
+
+    def forward(self, x):
+        return self.model(x)
+```
+
+### Step 4: **Define Train Loop**
+
+We define the train loop in the overrided `run` method of `LightningLite`, which is required by PyTorch Lightning.
+
+```python
+import os
+import torch
+
+data_path = os.environ.get("DATA_PATH", ".")
+batch_size = 256
+max_epochs = 3
+
+class Lite(LightningLite):
+    def run(self):
+        model = ResNet18(10, pretrained=False, include_top=False, freeze=True)
+        loss = nn.CrossEntropyLoss()
+        optimizer = torch.optim.Adam(model.parameters(), lr=0.01)
+        train_loader = create_dataloader(data_path, batch_size)
+
+        model, optimizer = self.setup(model, optimizer)
+        train_loader = self.setup_dataloaders(train_loader)
+        model.train()
+
+        for _i in range(max_epochs):
+            total_loss, num = 0, 0
+            for X, y in train_loader:
+                optimizer.zero_grad()
+                l = loss(model(X), y)
+                self.backward(l)
+                optimizer.step()
+                
+                total_loss += l.sum()
+                num += 1
+            print(f'avg_loss: {total_loss / num}')
+```
+
+### Step 5: **Run with Nano PyTorch LightningLite**
+
+```python
+Lite().run()
```

Comment:
change the interface to `nano().train()`? @yangw1234

Reply:
I am OK with `nano().train()`. But is it easier for users to use if we use the same name as `PyTorchLightning`?